compiler/rustc_error_codes/src/error_codes/E0208.md

```diff
@@ -0,0 +1,27 @@
+Error that only exists so that variance can be tested.
+
+Erroneous code example:
+
+```compile_fail,E0208
+#![feature(rustc_attrs)]
+
+#[rustc_variance]
+struct Test2<'a, 'b, 'c> { // ERROR [-, -, -]
+     x: &'a isize,
+     y: &'b [isize],
+     c: &'c str
+}
+
+fn main() {}
+```
+
+Below find a table mapping the symbol shown to the variance:
```

Comment:
There is really nothing more to be explained about this error code?

Reply:
Thanks for your feedback. What else should be explained about this error code?

Reply:
Try to adopt this point of view: you don't know what this error code is about at all. What information does the reader need to understand why they got this error?

In the current case, you only show a table of variance, but that's far from enough to understand what's going on in here.

Reply:
Thanks. A link to the following page would be enough? https://doc.rust-lang.org/nomicon/subtyping.html

Reply:
I have just realized it's already there

Reply:
I felt the link to the documentation was enough @mbartlett21 What else can I add here?